app/Console/Commands/InitCommand.php

```diff
@@ -101,7 +101,7 @@ public function handle(): void
     /**
      * Prompt user for valid database credentials and set up the database.
      */
-    private function setUpDatabase(): void
+    private function requestDatabaseCredentialsUpdate(): void
```

Comment:
I don't think this change is a good idea.

Reply:
Okay, I will revert it, but let me try to explain why:

"setUpDatabase" (and indeed the documentation's explanation of "set up the database"), to me, implies the process of actually setting up a database: creating the schema, running migrations, or whatnot. But this method isn't actually doing that. setUpDatabaseCredentials would at least be clearer, but it also isn't doing *that*, because it's actually requesting the credentials from the end user, and *then* setting them up.

This confused me, and I ended up wasting a bit of time until I read the entire method to determine what it actually did.